src/main/python/DetectLanguage.py

```diff
@@ -0,0 +1,23 @@
+# Archives Unleashed Toolkit (AUT):
+# An open-source platform for analyzing web archives.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from langdetect import detect
+
+def DetectLanguage(input: String):
+	if input == "":
+		return ""
```

Comment:
I wonder if a try ... catch would work better here.  It seems like there are a few different reasons why a langdetect might burp besides empty string.

https://github.com/shuyo/language-detection/blob/master/src/com/cybozu/labs/langdetect/LangDetectException.java